.github/workflows/binaries.yml

```diff
@@ -106,3 +106,24 @@ jobs:
         with:
           name: ${{ env.STELLAR_CLI_INSTALLER }}
           path: ${{ env.STELLAR_CLI_INSTALLER }}
+
+  cookbook:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup vars
+        run: |
+          version="$(cargo metadata --format-version 1 --no-deps | jq -r '.packages[] | select(.name == "stellar-cli") | .version')"
+          echo "VERSION=${version}" >> $GITHUB_ENV
+          echo "NAME=stellar-cli-${version}-docs-cookbook" >> $GITHUB_ENV
+
+      - name: Compress
+        run: |
+          tar czvf ${{ env.NAME }}.tar.gz cookbook FULL_HELP_DOCS.md
```

Comment:
The cookbook and full help are different things. The full help docs is referred to as the user manual in the docs, so I think we should rename the job to User Manual. We could rename the .md file to USER_MANUAL too maybe for consistency.

Reply:
Do you think we should upload FULL_HELP_DOCS.md and cookbook.tar.gz as separate artifacts?

Reply:
I think they should be separate. And named by what they contain. The user manual is at least maybe interesting to others.